ports/nethost/vcpkg.json

```
@@ -1,7 +1,6 @@
 {
   "name": "nethost",
-  "version-date": "2019-12-21",
-  "port-version": 2,
+  "version-date": "2022-05-30",
```

```suggestion
  "version": "6.0.5",
```
I think that since the upstream has updated the specific version number, `version-data` can be discarded. This will make subsequent version maintenance and updates more clear.